.gitignore

```diff
@@ -3,7 +3,10 @@
 .RData
 .Ruserdata
 .ipynb_checkpoints
+.idea
 data_der/
 data_raw/
 data_der.zip
+.DS_store
+*.pyc
 
```

Comment:
add .Rproj here for now and I can re-add the file

Reply:
I just did it and added Rproj back as well since I still had that saved on my computer, But I don't know how to update the pull request